CHANGELOG.md

```diff
@@ -5,13 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## dev
+## [3.2.2](https://github.com/nf-core/sarek/releases/tag/3.2.2) - Pierikjaure
```

Comment:
@asp8200 any idea for a release codename?

Reply:
Vuoinesluobbalah?

https://goo.gl/maps/sCnN4XfpTRPxStTd6

Reply:
Gällivare?

https://goo.gl/maps/yVERsgxd1tR7VHny9

Reply:
I like `Vuoinesluobbalah`

Reply:
It rolls nicely off the tongue